roles/mongodb_linux/files/thp-enable.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Enable Transparent Huge Pages
+ConditionPathIsDirectory=/sys/kernel/mm/transparent_hugepage
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/env sh -c "echo 'always' > /sys/kernel/mm/transparent_hugepage/enabled && echo 'always' > /sys/kernel/mm/transparent_hugepage/defrag"
```

Comment:
I'm not sure if this was intentional, but this unit is quite different from the example [provided in the MongoDB docs](https://www.mongodb.com/docs/manual/administration/tcmalloc-performance/#create-the-systemd-unit-file).

Reply:
This role was created a very long time ago... I suspect the recommendation has changed in the meantime. I will update it to use the example from the docs.